<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INBNRMARITA" failures="1" tests="14" name="Admin- PO creation" time="4331.856" errors="0" timestamp="2021-11-10T20:35:50 IST">
  <testcase classname="script.CreatePO" name="buyerLinkValidation" time="4.971"/>
  <testcase classname="script.CreatePO" name="buyerOnboardingPage" time="1.933"/>
  <testcase classname="script.CreatePO" name="searchUser" time="2680.305"/>
  <testcase classname="script.CreatePO" name="userCardValidation" time="0.064"/>
  <testcase classname="script.CreatePO" name="validateCategory" time="19.606"/>
  <testcase classname="script.CreatePO" name="validateQuantity" time="490.449"/>
  <testcase classname="script.CreatePO" name="validateUnit" time="4.525"/>
  <testcase classname="script.CreatePO" name="validatePrice" time="985.834"/>
  <testcase classname="script.CreatePO" name="addCrop" time="0.121">
    <failure type="java.lang.AssertionError" message="expected [ Crop Added Successfully] but found [Crop Added Successfully]">
      <![CDATA[java.lang.AssertionError: expected [ Crop Added Successfully] but found [Crop Added Successfully]
at script.CreatePO.addCrop(CreatePO.java:269)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- addCrop -->
  <testcase classname="script.CreatePO" name="poDates" time="9.044"/>
  <testcase classname="script.CreatePO" name="deliveryDeliveryValidation" time="18.592"/>
  <testcase classname="script.CreatePO" name="cancelPoValidation" time="4.294"/>
  <testcase classname="script.CreatePO" name="generatePO" time="5.189"/>
  <testcase classname="script.CreatePO" name="poCreation" time="89.428"/>
</testsuite> <!-- Admin- PO creation -->
